src/Core/Utils/Index.hpp

```diff
@@ -32,6 +32,8 @@ class RA_CORE_API Index
     /// INVALID
     constexpr bool isInvalid() const;
     constexpr void setInvalid();
+
+    /// @brief return an invalid Index
```

Comment:
It seems that our guidelines expect `\` as mark for doxygen keyword (e.g. `\brief`). see, e.g. ControlPanel.hpp below. I remember discussions in the steering committee about that but find nothing in the coding conventions included in the documentation.

Reply:
Right, I'll add it to the code convention (and try to automatically replace most of them)